docs/src/development/ssh.md

```diff
@@ -2,131 +2,118 @@
 title: "Using SSH"
 weight: 12
 description: |
-  One of the ways Platform.sh keeps things secure is by using SSH behind the scenes. Users can interact with their environment through a command shell, or push changes to the environment's Git repository, and both of these features rely on SSH.
+  Platform.sh uses Secure Shell (SSH) to help keep things secure. You can interact with your deployed environments or use the Platform.sh CLI, and both of these features rely on SSH.
 sidebarTitle: "SSH"
 ---
 
 {{< description >}}
 
-Secure Shell Protocol, SSH, supports certificate-based and keypair-based authentication.  Certificate-based authentication is faster to set up and generally easier to use, provided you have a web browser available on your computer.  Alternatively, you may use keypair-based authentication if you are setting up an automation tool, or simply prefer that method.
+This means you can do things like securely log in to your deployed server to troubleshoot and see what's different than your local environment or read logs. And push changes to your Git repository. All secured through SSH.
```

Comment:
```suggestion
This means you can securely log in to your deployed server to troubleshoot and see what's different than your local environment or read logs. And push changes to your Git repository. All secured through SSH.
```
I guess I'm more OK with things in this context if you do feel they add value as a way to say there are more, so feel free to ditch the comment :)

Reply:
I didn't like "things", just wanted to keep the idea that there's more. But there has to be a better way to say it. :thinking: